tests/test_functional.py

```diff
@@ -316,3 +316,10 @@ def test_obal_release_copr_hello():
         "copr-cli watch-build"
     ]
     assert_mockbin_log(expected_log)
+
+
+@obal_cli_test(repotype='upstream')
+def test_obal_bump_release_hello():
+    assert_obal_success(['bump-release', 'hello'])
+
+    assert subprocess.check_output(['rpmspec', '-q', '--queryformat=%{release}', '--srpm', 'packages/hello/hello.spec'])  # noqa: E501
```

Comment:
Should we assert a new value? `== '2'` or something?

Reply:
I think I would like that, yeah